enterprise/admin/views.py

```diff
@@ -291,6 +291,13 @@ def _handle_bulk_upload(cls, enterprise_customer, manage_learners_form, request,
 
         return all_processable_emails
 
+    # TODO: this method is HUGE (> 100 lines) and complicated. Refactor it.
```

Comment:
When will we address this TODO?  Will it be part of this PR?  Will we log a ticket to revisit after a merge?